week-2/implement/cases.js

```diff
@@ -10,8 +10,17 @@
 // Then it returns the string in UPPER_CAMEL_CASE, so "HELLO_THERE"
 
 // Test case: we expect "lord of the rings" to be "LORD_OF_THE_RINGS"
-// Test case: we expect "the great gatsby" to be "THE_GREAT_GATSBY"
-// Test case: we expect "the da vinci code" to be "THE_DA_VINCI_CODE"
-
 // Come up with a clear, simple name for the function
 // Use the string documentation to help you plan your solution
+
+function camelCaseToWords(text) {
```

Comment:
This is a great little function, well done. For next time, I'd probably name it differently, functions rend to get named something g like convertToUpperCamelCase() but this is a mere niggle